### PR TITLE
Created note deletion page UI

### LINKED ISSUE #2: Create note deletion page UI
- [x] Add modal showing deletion prompt

### PR DESCRIPTION
**Pull request by sreerag-ms**
Fixes #2 

* Added deletion of Note.
* Shows a prompt to confirm the deletion.

### CODE REVIEW

File: app/javascript/src/components/Common/DeletePrompt/index.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+);
+
+DeletePrompt.propTypes = {
+  showPrompt: PropTypes.bool.isRequired,
+  onDelete: PropTypes.func.isRequired,
+  onCancel: PropTypes.func.isRequired,
+  title: PropTypes.string,
+  message: PropTypes.string,
+};
+export default DeletePrompt;
```
Comment (karthiknmenon) [inline app/javascript/src/components/Common/DeletePrompt/index.jsx:36]: We don't need this. Please use [Alert](https://neetoui-v2.netlify.app/?path=/story/overlays-alert--alert-story) from neetoUI.

File: app/javascript/src/components/Notes/Card.jsx [hunk 2]
```diff
@@ -16,7 +24,14 @@ const Card = ({ title, description, user, createdAt }) => {
         <Typography style="h4">{title}</Typography>
         <Dropdown icon={MenuVertical} buttonStyle="text">
           <li>Edit</li>
-          <li>Delete</li>
+          <li
+            onClick={() => {
+              setSelectedNote(id);
+              setShowDeletePrompt(true);
```
Comment (karthiknmenon) [inline app/javascript/src/components/Notes/Card.jsx:30]: `setIsDeleteAlertOpen()`

File: app/javascript/src/components/Notes/index.jsx [hunk 3]
```diff
@@ -1,17 +1,40 @@
 import React, { useState } from "react";
 
+import { Toastr } from "@bigbinary/neetoui/v2";
```
Comment (karthiknmenon) [inline app/javascript/src/components/Notes/index.jsx:3]: Please use aliases.

File: app/javascript/src/components/Notes/index.jsx [hunk 4]
```diff
@@ -1,17 +1,40 @@
...
+
 import Card from "./Card";
 import { SIDE_MENU_ITEMS, NOTES_DATA } from "./constants";
 
-import SideMenu from "../Common/SideMenu";
-import TitleBar from "../Common/TitleBar";
-
 const Notes = () => {
   const [showSideMenu, setShowSideMenu] = useState(true);
+  const [showDeletePrompt, setShowDeletePrompt] = useState(false);
```
Comment (karthiknmenon) [inline app/javascript/src/components/Notes/index.jsx:14]: `const [isDeleteAlertOpen, setIsDeleteAlertOpen] = useState(false);`

**Review by karthiknmenon — COMMENTED** [review]
@sreerag-ms Please check the suggestions.

File: app/javascript/src/components/Common/DeletePrompt/index.jsx (hunk 1 above)
Comment (sreerag-ms) [reply app/javascript/src/components/Common/DeletePrompt/index.jsx:36]: Did this as there was no way to customize the style of the button in the neetoUI alert, to the one shown in the Figma design.

**Pull request merged**